ccp-project/portus
# pyportus: Switch to maturin instead of setuptools-rust

To test:
1. make a virtual environment
2. maturin develop
3. cd examples && sudo ./\<venv\>/python aimd.py (set the RUST_LOG environment variable to see logging)

To install:
1. maturin build
2. pip install the resulting .whl
